### PR TITLE
Update README.md Docker Commands and default udp port

### PR DESCRIPTION
**Pull request by Mondanzo**
Fixed Docker Commands not lining up with the .env.example

### 📓 Description
Updates the readme with appropiate docker commands for port forwarding.

### ☑️ Checklist

- [x] Documentation is up to date
- [x] Versions are bumped as needed

### 🔗 Related issues
https://github.com/foxssake/netfox/issues/218

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -36,7 +36,7 @@ NORAY_UDP_RELAY_TIMEOUT=30s
 NORAY_UDP_RELAY_CLEANUP_INTERVAL=30s
 
 # Port where noray listens for UDP relay requests from hosts
-NORAY_UDP_REGISTRAR_PORT=8809
+NORAY_UDP_REGISTRAR_PORT=8890
```
Comment (elementbound) [inline .env.example:39]: Why this change? The default registrar port is 8809 as can be seen [here](https://github.com/foxssake/noray/blob/0db49adb4be3fcbebcfb885d7823fd56279626cb/src/config.mjs#L38).

File: README.md [hunk 2]
```diff
@@ -75,14 +75,16 @@ Build and run docker:
 
 ```
 docker build . -t noray
-docker run -p 8090:8090 -p 8091:8091 --env-file=.env -t noray
+docker run -p 8890:8890 -p 8890:8890/udp -p 8091:8091 --env-file=.env -t noray
```
Comment (elementbound) [inline README.md:78]: Based on default config, noray listens on 8890 TCP for commands, 8891 UDP for registrar, and 8891 HTTP for exposing metrics.

```suggestion
docker run -p 8890:8890 -p 8891:8891/udp -p 8891:8891 --env-file=.env -t noray
```

File: README.md [hunk 3]
```diff
@@ -75,14 +75,16 @@ Build and run docker:
...
 ```
 
 Or run prebuilt docker:
 ```
-docker run -p 8090:8090 -p 8091:8091 --env-file=.env -t ghcr.io/foxssake/noray:main
+docker run -p 8890:8890 -p 8890:8890/udp -p 8091:8891 --env-file=.env -t ghcr.io/foxssake/noray:main
```
Comment (elementbound) [inline README.md:83]: Same as above.

```suggestion
docker run -p 8890:8890 -p 8891:8891/udp -p 8891:8891 --env-file=.env -t ghcr.io/foxssake/noray:main
```

File: README.md [hunk 4]
```diff
@@ -75,14 +75,16 @@ Build and run docker:
...
 ```
 
+_Please Note: This configuration will only allow NAT Punching but no port forwarding. Make sure to add `-p 49152-51200:49152-51200/udp` to allow all default udp relay ports. Beware! This will increase the deploy time severely!_
```
Comment (elementbound) [inline README.md:86]: I'd prefer to call it relaying, just to stay consistent with the existing docs.

```suggestion
_Please Note: This configuration will only allow NAT punchthrough but no relaying. Make sure to add `-p 49152-51200:49152-51200/udp` to allow all default udp relay ports. Beware! This will increase the deploy time severely!_
```

**Review by elementbound — COMMENTED** [review]
Thanks for raising all these! It seems there were indeed issues with the README. Added some notes. Could you also please bump the version in package.json from 1.4.2 to 1.4.3 please? 

Also sorry for the late review, initially the diff showed full files instead of just a few lines changed, and then got carried away with other stuff.

File: .env.example (hunk 1 above)
Comment (Mondanzo) [reply .env.example:39]: The idea behind the change was to use the same port number as TCP to avoid having varying protocol ports open across different ports. Made it for me just easier to be aware of which ports are actually open and not feel like "wasting" any ports. I didn't get too deep into the code when I made that change to see there is a default option set, in hindsight that makes a lot of sense.

**Comment by Mondanzo** [comment]
I think at this point it would be easier to open a new pull request from the main branch, I kinda neglected the patch-1 branch and didn't intend it to also include the error handling originally.

**Comment by elementbound** [comment]
PR looks good, I plan to add some finishing touches and merge. I've run into some weird issue locally where the connection doesn't go through for some reason ( as opposed to the tomfol.io instance ). Will merge once I figure that out.

**Comment by elementbound** [comment]
Thanks for the PR @Mondanzo, merged!
